examples/cookbook/prompt_engineering/chaining_based/diverse/diverse.py

```diff
@@ -88,7 +90,7 @@ async def diverse(query: str, num_variations: int):
     return best_response
 
 
-async def run_self_consistency(prompt, num_variations=3):
+async def run_self_consistency(prompt: Any, num_variations: int = 3) -> Coroutine:
```

Comment:
same thing here re: not needing `Coroutine` definintion for `async` marked function?